test/webapi/test_s3buckethandlers.py

```diff
@@ -27,7 +27,7 @@ class S3BucketHandlersTest(unittest.TestCase):
 
     @unittest.skipUnless(XCUBE_SERVER_IS_RUNNING, SKIP_HELP)
     def test_open_cube_from_xube_server(self):
-        ds = open_cube('s3bucket/local', format_name='zarr', endpoint_url=SERVER_URL)
+        ds = open_cube('s3bucket/local', format_name='zarr', client_kwargs={'endpoint_url':SERVER_URL})
```

Comment:
This makes it harder for users to use `open_cube`. I'd prefer to keep `endpoint_url` and add it to `client_kwargs` internally.